conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosNodeFetchers.scala

```diff
@@ -545,37 +545,6 @@ private[tezos] trait TezosBlocksDataFetchers {
     }
   }
 
-  /** a fetcher of baker rolls for blocks */
-  implicit val bakersFetcher = new FutureFetcher {
```

Comment:
I merged the usage into a single fetcher.

Previously we fetched bakers both using blocks and block hashes as input. Now we re-use the more specific one with an adapter for the other case.
After all if you have a block, you can always extract the hash and call the fetcher based on the hash.

The corollary is that you need to adapt the output, because a fetcher gives you batches of results indexed on the input value (you get back a sequence of input paired with the output). So you might need to find the original block value associated with each hash, to have a block/bakers pair, instead of hash/bakers.

That's what I've done with `adaptResults`, here: https://github.com/Cryptonomic/Conseil/pull/839/files/76109ec1a5cb373bdbd9010608871ced6e9ce477#diff-a6b43db1952e44cb604adee47f25e528R364